install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -73,11 +73,11 @@ spec:
       rules:
         - alert: MCDPivotError
           expr: |
-            mcd_pivot_err > 0
+            mcd_pivot_err_total > 0
```

Comment:
Same remark here, you'd usually use `rate()` or `increase()` to alert on a counter metric but not having much knowledge about MCD, it's hard for me to be more specific.

Reply:
behavior should be same as previous. After pivot (OSUpdate), MCD will perform node reboot.

Reply:
As per further conversation, have updated PR with use of increase()